### DISCUSSION TITLE
Can't get webserver to work no mater what.

### DISCUSSION THREAD

**Discussion by Hellmanx** [plan-player-analytics/Plan#3050] [Q&A] [answered]
I have a fresh install of plan.  Plan-5.5-build-2391 specifically.  I cannot get to the web server.

Port: 8804 as default.
HTTP and HTTPS will not work.
Firewall is off.

Error when trying to access webserver on any machine including localhost: 
[SEVERE] [/10.0.0.10:42727] <-> InitialHandler - encountered exception: net.md_5.bungee.util.QuietException: Unexpected packet received during login process! 4554202f20485454502f312e310d0a48

At a complete loss.

**Comment by AuroraLS3** [comment]
> Unexpected packet received during login process! 

This happens if you put the Minecraft (the game) address and port in the browser instead of the Plan address and port

**Reply by Hellmanx** [reply]
Definitely typing in port 8804.
I know the MC servers do not conflict port wise.  They're around 24000+

**Reply by AuroraLS3** [reply]
There's some connection issue tips here https://github.com/plan-player-analytics/Plan/wiki/Bukkit-Set-Up#step-5-try-to-access-the-page (the flowchart for example)

**Reply by Hellmanx** [reply]
It's `ERR_CONNECTION_REFUSED`.  Not really much troubleshooting I can do other than look at the config and confirm the port number.

I don't see any obvious setting that will turn the web server on or off other than `Enabled: false`.  But that's under `Alternative_IP:` I have no clue what that'd be used for.  But even if I set Enabled as 'True' same issue.

**Reply by AuroraLS3** [reply]
Here's the access issue flowchart that usually solves things

![](https://cdn.discordapp.com/attachments/661865988643356672/825722262534750238/Plan_website_access_issue_flowchart.jpg)

**Reply by Hellmanx** [reply]
1. Yes
2. Yes
3. No - Never heard of it
4. No
5. Yes (Should be enabled properly, it's default)
6. Firewall is off, so shouldn't be an issue.
7. Firewall is off.

Has plan enabled properly?
I will get an SQL error, but I do not think it's necessary for the webserver to function.  Maybe it is?
22:59:09 [SEVERE] [Plan] ----------------------------------------
22:59:09 [SEVERE] [Plan] Error: Failed to start MySQL: Communications link failure. Plan could not connect to MySQL-
- Check that database address 'localhost:3306' accessible.
- Check that database called 'Plan' exists inside MySQL.
- Check that MySQL user 'root' has privileges to access the database.
- Check that other MySQL settings in Plan config correct.
  More help: https://github.com/plan-player-analytics/Plan/wiki/Bungee-Set-Up#step-2-create-a-mysql-database-for-plan
22:59:09 [SEVERE] [Plan] ----------------------------------------
22:59:09 [SEVERE] [Plan] Plugin Failed to Initialize Correctly. If this issue is caused by config settings you can use /planbungee reload

**Reply by AuroraLS3** [reply, marked as answer]
Yep follow those instructions to fix the Mysql connection problem
